includes/Maps_DisplayMapRenderer.php

```diff
@@ -145,6 +145,13 @@ protected function handleMarkerData( array &$params, Parser $parser ) {
 
 		if ( $params['mappingservice'] === 'openlayers' ) {
 			$params['layers'] = self::evilOpenLayersHack( $params['layers'] );
+		} else if ( $params['mappingservice'] === 'leaflet' ) {
+			global $egMapsLeafletLayerDependencies;
+			$layerDependencies = [];
+			$layerDependencies[] = "<script src='https://open.mapquestapi.com/sdk/leaflet/v2.2/mq-map.js?key=" . $GLOBALS['egMapsLeafletLayersApiKeys']['MapQuestOpen'] . "'></script>";
+			MapsMappingServices::getServiceInstance( 'leaflet' )->addLayerDependencies(
+				array_unique( $layerDependencies )
+			);
```

Comment:
Why are you putting this code here? It seems to have little to do with the method and not need anything from it